eng/targets/Imports.targets

```diff
@@ -175,8 +175,8 @@
   </ItemGroup>
 
   <ItemGroup>
-    <!-- Include BannedSymbols covering all projects -->
-    <AdditionalFiles Include="$(MSBuildThisFileDirectory)..\config\BannedSymbols.txt" />
+    <!-- Include BannedSymbols covering all product projects -->
+    <AdditionalFiles Include="$(MSBuildThisFileDirectory)..\config\BannedSymbols.txt" Condition="'$(IsTestProject)' == 'false'" />
```

Comment:
This seems a rather big change for this. Should we be having multiple lists, depending on whether it's a test project or not? Presumably some of the existed banned apis should still not be called in tests?

Reply:
We already have banned API list specific for tests: `eng\config\test\BannedSymbols.txt`
Banned product public APIs should actually be still tested so there seems no point in disallowing them in tests, imo.